### DISCUSSION TITLE
v2.2.9 release concern...

### DISCUSSION THREAD

**Discussion by brantmessenger** [livewire/livewire#1817] [1. Help] [answered]
Was working through an "emitTo" issue with the event being received by other components. Seen the change to address this and did a composer update. 
https://github.com/livewire/livewire/commit/3c068addb7ff7c45cac1ab14e31d7eea03759807
I have version 2.2.9 but the code has NOT been replaced.

https://user-images.githubusercontent.com/1199872/96034419-20b24700-0e27-11eb-8029-1a5e6cf65439.png

To confirm, I pulled down the zip of the release and sure enough `->to($name)` is there instead of `->component($name)`.

Class: `/livewire-2.2.9/src/ComponentConcerns/ReceivesEvents.php`
Method: `emitTo`

Can anyone give me insight to what I might be doing wrong? Or is this actually a source issue?

**Comment by calebporzio** [comment, marked as answer]
Thanks, this IS an issue - it was PRed and will be fixed in the upcoming release. Thanks!
